crates/proposer-client/src/network_prover.rs

```diff
@@ -2,37 +2,52 @@ use std::time::Duration;
 
 use alloy_primitives::B256;
 use anyhow::{Context, Error};
-use sp1_sdk::{NetworkProver, SP1ProofWithPublicValues};
+use sp1_sdk::{
+    NetworkProver, Prover, SP1ProofWithPublicValues, SP1VerificationError, SP1VerifyingKey,
+};
 
 /// This prover waits for the SP1 cluster generated
 /// AggSpanProof based on the proof id.
 #[tonic::async_trait]
-pub trait AggSpanProver {
+pub trait AggregatedProver {
```

Comment:
missing rename in the rustdoc right above

Reply:
There is a slight terminology inconsistency. Sometimes it is an "AggregationProof" and sometimes "AggregatedProof".

Reply:
@hadjiszs @iljakuklic Lets settle then with `AggregationProof`?